js/script.js

```diff
@@ -10,18 +10,17 @@ document.addEventListener('DOMContentLoaded', function() {
 });
 
 var prepareTweet = function(input) {
-  var data = input[0];
+  if (Array.isArray(input)) { var data = input[0]; }
```

Comment:
when data wouldn't be an array - data will be undefined ;) 

`element.innerHTML = data.content;` what will this return?